UITests/LoginManagerTests.swift

```diff
@@ -721,92 +720,4 @@ class LoginManagerTests: KIFTestCase {
 
         closeLoginManager()
     }
-    
-    func testLoginsListPromptsForPasscodeOnReentryFromBackground() {
```

Comment:
we don't need these tests anymore?

Reply:
yes, those were actually in XCUITests for a while, but I realized that they haven't been deleted.